app.py

```diff
@@ -44,17 +64,21 @@ def upload_file():
         filename = secure_filename(file.filename)
         file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
         file_in = glob.glob('./uploads/*.json')
-        if (convert_from == "json" and convert_to == "png"):
-            os.system('python ./converters/' + convert_from + '2' + convert_to + '.py -i ' + file_in[0])
-        else:
-            json2glm()
+        dict_args['file_in'] = file_in
+        try:
+            supported_from_to_conversions[convert_from][convert_to](dict_args)
+        except:
```

Comment:
What exception are you catching here? Be specific.
Also, after you catch the exception you just print a statement and you are letting the flow of execution continue. Why?

Reply:
now I'm flashing "conversion failed" and returning to the same page